test/physics/em/PhotoelectricInteractor.test.cc

```diff
@@ -113,10 +113,8 @@ class PhotoelectricInteractorTest
         }
 
         // Check conservation between primary and secondaries
-        // TODO: atomic relaxation has not been implemented. If the binding
-        // energy of the sampled shell is larger than the incident photon
-        // energy, or there is an energy balance following atomic deexcitation,
-        // the energy will be deposited locally.
+        // Since momentum is transferred to the atom, we don't expect it to be
+        // conserved between the incoming and outgoing particles
```

Comment:
@vrpascuzzi Did you end up separating the momentum and energy conservation checks? We could add that to this PR.

Reply:
@sethrj,

You mentioned doing this in such a way that we wouldn't need to make modifications to existing tests that call `check_conservation`. My initial thought was to change the function definition:
```
void check_conservation(const Interaction& interaction) const;
```
to
```
void check_conservation(const Interaction& interaction, bool do_energy, bool do_momentum) const;
```
But maybe we could include conservation information directly into the `Interaction` class?

Reply:
OK, my thought was initially we split up the existing `check_conservation` into two functions, and leave the original  signature for compatibility and simplicity:
```c++
void check_conservation(const Interaction& interaction) const
{
    this->check_energy_conservation(interaction);
    this->check_momentum_conservation(interaction);
}
```
But what I'm suggesting is that if you haven't done that, we could add it to this PR.

Reply:
Ah, gotcha! Makes sense.

I've done this but it's fine to add to this PR is fine -- it's a simple change.

Reply:
Thanks @vrpascuzzi! I'll go ahead and add it since I need it for the Livermore interactor.

Reply:
I'll leave that to you and @amandalund -- I'm OK with merging this branch, and then the BetheHeitler, and then cleaning up and checking whatever conservation we can at that point.

Reply:
Ok that works!